components/article/ArticleList.tsx

```diff
@@ -1,6 +1,11 @@
 import { useRouter } from "next/router";
```

Comment:
Liking should be handled in the `articles/[pid].tsx` file and should look like the following
<img width="964" alt="Screen Shot 2020-06-26 at 9 52 13 AM" src="https://user-images.githubusercontent.com/35946905/85881667-d97a4a80-b792-11ea-8409-7925cd993416.png">